app/models/thredded/messageboard.rb

```diff
@@ -6,7 +6,8 @@ class Messageboard < ActiveRecord::Base
                 use:            [:slugged, :reserved],
                 # Avoid route conflicts
                 reserved_words: ::Thredded::FriendlyIdReservedWordsAndPagination.new(
-                  %w(messageboards preferences private-topics autocomplete-users theme-preview admin)
+                  %w(messageboards preferences private-topics private-posts posts autocomplete-users
+                     theme-preview admin)
```

Comment:
This array has gotten big enough where it's a little bit of mental overhead to parse it on a quick look. Could you break these out into their own lines? It'd make diffs cleaner too.